src/session/SessionManager.h

```diff
@@ -40,10 +45,10 @@ struct SessionStats {
 	uint64_t num_packets;
 };
 
-class NetSessions {
+class SessionManager final {
```

Comment:
With this now being namespaced, can skip the `Session` prefix, so just `Manager` seems fine. Likewise for a couple of other classes, e.g., `SessionKey`.